src/community/jdbcstore/src/main/resources/jdbcstore.properties

```diff
@@ -44,4 +44,4 @@ pool.validationQuery=SELECT now()
 deleteDestinationOnRename=true
 
 # leave the following directories on the filesystem -- no spaces!
-ignoreDirs=data,jdbcstore,workspaces,layergroups,temp,tmp
+ignoreDirs=data,jdbcstore,workspaces,layergroups,temp,tmp,logs
```

Comment:
I think for now workspaces should be removed from the ignored directories.
better to store too much in the db than too little, the config files done by jdbcconfig will be ignored (unless jdbcconfig is turned off, in which case jdbcstore will be doing the catalog as well). I'm going to change this.